e2e-tests/playwright/e2e/plugins/github-issues/github-issues.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { expect, test } from "@playwright/test";
+import { UIhelper } from "../../../utils/ui-helper";
+import { Common } from "../../../utils/common";
+
+// Test GitHub Issues plugin functionality
+test.describe("GitHub Issues Plugin", () => {
+  const showcase = "Backstage Showcase";
+
+  let uiHelper: UIhelper;
+  let common: Common;
+
+  test.beforeEach(async ({ page }) => {
+    test.setTimeout(180000);
```

Comment:
Some timeout values, such as test.setTimeout(180000) (3 minutes), seem excessively high. It might be worth reviewing whether these durations are absolutely necessary.

Reply:
done, was decreased to 60000